bundle/pom.xml

```diff
@@ -67,21 +67,20 @@
                         <Import-Package>
                             !org.apache.myfaces.*,
                             com.google.inject;version="[1.0.0, 2.0.0)";resolution:=optional,
-                            javax.annotation,
+                            jakarta.annotation,
                             javax.crypto,
                             javax.crypto.spec,
-                            javax.ejb;resolution:=optional,
-                            javax.el;version="[1.0.0, 3.0.0)",
+                            jakarta.ejb;resolution:=optional,
+                            jakarta.el;version="[5.0.0, 6.0.0)",
                             javax.naming,
-                            javax.persistence;version="[1.0.0, 3.0)";resolution:=optional,
-                            javax.portlet;version="[1.0.0, 2.1)";resolution:=optional,
-                            javax.servlet;version="[3, 5)",
-                            javax.servlet.http;version="[3, 5)",
-                            javax.servlet.jsp;version="[2.1.0, 3.1)",
-                            javax.servlet.jsp.jstl.core;version="[1.1.2, 2.0.0)",
-                            javax.servlet.jsp.tagext;version="[2.1.0, 3.1)",
-                            javax.servlet.annotation;version="[3, 5)";resolution:=optional,
-                            javax.validation.*;version="[1.0.0, 2.0.0)";resolution:=optional,
+                            jakarta.persistence;version="[3.1.0, 4.0.0)";resolution:=optional,
+                            jakarta.servlet;version="[6.0.0, 7.0.0)",
+                            jakarta.servlet.http;version="[6.0.0, 7.0.0)",
+                            jakarta.servlet.jsp;version="[6.0.0, 7.0.0)",
+                            jakarta.servlet.jsp.jstl.core;version="[3.0.0, 4.0.0)",
+                            jakarta.servlet.jsp.tagext;version="[3.1.0, 4.0.0)",
```

Comment:
Should I remove  `jakarta.servlet.jsp` , `jakarta.servlet.jsp.jstl.core`, and `jakarta.servlet.jsp.tagext` ? 

There are no hits for these packages in the 4.0 code base.  Only hits I see are in `extensions/quarkus/runtime/pom.xml`

Reply:
Yep see: https://github.com/apache/myfaces/pull/316

Reply:
I don't see them in Mojarra either.  Will remove then :) 


Mojarra (org.glassfish javax.faces 4.0.0) Imports:

 jakarta.el;version="[5.0,6)",

 jakarta.servlet.annotation;resolution:=optional;version="[6.0,7)",
 jakarta.servlet;version="[6.0,7)",
 jakarta.servlet.http;version="[6.0,7)",

 jakarta.validation;resolution:=optional;version="[3.0,4)",
 jakarta.validation.groups;resolution:=optional;version="[3.0,4)",

  jakarta.annotation;version="[2.1,3)",

  jakarta.ejb;resolution:=optional;version="[4.0,5)",

  jakarta.persistence;resolution:=optional;version="[3.1,4)",